drivers/doc.txt

```diff
@@ -43,3 +43,10 @@
  * @ingroup     drivers
  * @brief       Drivers for storage devices
  */
+
+/**
+ * @defgroup    drivers_can CAN Drivers
+ * @ingroup     drivers
+ * @ingroup     can
```

Comment:
For simplicity, we can keep drivers_netdev, but the displayed name can become "Communication Device Drivers".
> After all, CAN stands for Controller Area Network ;)

Sure!